assets/src/apps/authoring/components/PropertyEditor/custom/TorusImageBrowser.tsx

```diff
@@ -6,7 +6,14 @@ import { selectProjectSlug } from '../../../store/app/slice';
 import { MediaPickerModal } from '../../Modal/MediaPickerModal';
 import { MediaBrowserComponent, TorusMediaBrowserWrapper } from './TorusMediaBrowserWrapper';
 
-const _TorusImageBrowser: MediaBrowserComponent = ({ id, label, value, onChange, onBlur }) => {
+const _TorusImageBrowser: MediaBrowserComponent = ({
+  id,
+  label,
+  value,
+  onChange,
+  onBlur,
+  onFocus,
```

Comment:
Since we are adding the media uploader widget, we need to handle the 'back space causing the delete window pop-up' issue (https://github.com/Simon-Initiative/oli-torus/pull/5280). So, I have added the 'onFocus' event handler to the component.